pkg/controller/volume/persistentvolume/index.go

```diff
@@ -181,10 +181,29 @@ func (c byAccessModes) Len() int {
 	return len(c.modes)
 }
 
+func claimToVolumeKey(claim *v1.PersistentVolumeClaim) string {
```

Comment:
For methods claimToVolumeKey() and ClaimToClaimKey(),  should "== v1.TenantDefault" be "== v1.TenantSystem"?

Reply:
In fact, since v1.PersistentVolumeClaim is a namespaced object, claim.tenant should never be empty in current implementation.

Reply:
correct. 

the check for claim.Tenant == v1.TenantDefault is to be (backward) compatible with the MetaNamespaceKeyFunc() function in store.go which is used in many many places.

Reply:
Actually, I think we need a big change to remove metav1.TenantDefault and replace it with metav1.TenantSystem. Metav1.TenantDefault is something we had before the notion of tenant world came into being.